code/messaging/src/phone/java/com/adobe/marketing/mobile/InAppNotificationHandler.java

```diff
@@ -115,7 +115,7 @@ void fetchMessages() {
      */
     void handlePersonalizationPayload(final Map<String, Object> payload) {
         if (MessagingUtils.isMapNullOrEmpty(payload)) {
-            Log.warning(LOG_TAG, "%s - Empty content returned in call to retrieve in-app messages.", SELF_TAG);
+            Log.debug(LOG_TAG, "%s - Empty content returned in call to retrieve in-app messages. Clearing local cache.", SELF_TAG);
```

Comment:
`Clearing local cache.` is scary 😨 

maybe `Clearing local cache for in-app messages.`?

Reply:
i agree, will update